src/ConfigurationTrait.php

```diff
@@ -92,10 +92,19 @@ public function getConfig()
                     'port' => isset($config['port']) ? $config['port'] : null,
                     'name' => $config['database'],
                     'charset' => isset($config['encoding']) ? $config['encoding'] : null,
-                    'unix_socket' => isset($config['unix_socket']) ? $config['unix_socket'] : null,
                 ]
             ]
-        ]);
+        ];
+
+        $mysqlAttributesArray = [ 'ssl_ca', 'ssl_cert', 'ssl_key' ];
```

Comment:
No need to suffix it with "Array". ``$mysqlAttributes`` is enough.
The space before the first element and the last element are not needed as well.